resources/js/features/events/components/BoxArtImage/BoxArtImage.tsx

```diff
@@ -0,0 +1,22 @@
+import type { FC } from 'react';
+import { useTranslation } from 'react-i18next';
+
+interface BoxArtImageProps {
+  event: App.Platform.Data.Event;
+}
+
+export const BoxArtImage: FC<BoxArtImageProps> = ({ event }) => {
+  const { t } = useTranslation();
+
+  if (!event.legacyGame?.imageBoxArtUrl || event.legacyGame.imageBoxArtUrl.includes('000002')) {
+    return null;
+  }
+
+  return (
+    <img
+      className="max-w-full rounded-sm"
+      src={event.legacyGame.imageBoxArtUrl}
+      alt={t('boxart')}
+    />
```

Comment:
Should this pop up a full screen image like the main media images?

Reply:
I think it should. We may get some complaints about text legibility. Supporting higher-res imagery is something we'll need to figure out at some point in the future. Updated in latest.

![Screenshot 2025-02-24 at 8 04 14 PM](https://github.com/user-attachments/assets/2e4d78bc-8e4d-4d99-9983-9e1f2432a362)